hs_app_netCDF/models.py

```diff
@@ -127,10 +127,7 @@ def __unicode__(self):
 
     @classmethod
     def remove(cls, element_id):
-        variable = Variable.objects.get(id=element_id)
-        if Variable.objects.filter(object_id=variable.object_id, content_type__pk=variable.content_type.id).count()== 1:
-            raise ValidationError("The only variable of the resource can't be deleted.")
-        variable.delete()
+        raise ValidationError("The variable of the resource can't be deleted.")
```

Comment:
@ironfroggy  The extended metadata elements of variable is not allowed to be deleted by users. So the only way to delete all the variables is to delete the whole resource. 

The AbstractResource class has the method "delete()". This method will delete the resource and also its resource files and all the metadata elements. (hs_core\models.py line 1079). As NetCDF resource is its subclass, I think it can automatically delete the variable elements when the resource is deleted.

@pkdash would you please comment on whether I need to add code to delete the metadata elements? Thank you